certified-connectors/Power Textor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6782 @@
+                    }
+                  },
+                  "description": "Data"
+                },
+                "Success": {
+                  "type": "boolean",
+                  "description": "Indicates whether action run successfully or not. True if action run successfully otherwise false.",
+                  "title": "Success",
+                  "enum": [true, false]
+                },
+                "Error": {
+                  "type": "object",
+                  "properties": {
+                    "Status": {
+                      "type": "string",
+                      "description": "Sent status of the message.",
+                      "title": "Status"
+                    },
+                    "Message": {
+                      "type": "string",
+                      "description": "Body of the message.",
+                      "title": "Message"
+                    }
+                  },
+                  "description": "Error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Schedule review text (SMS) message for multiple contacts",
+        "description": "Schedule review messages in bulk for Power Textor contacts by specifying date and time on which the messages should be sent. Send pin location of your business or service area within the text message.",
+        "operationId": "ScheduleReviewTextMessageForContacts",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "To": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "To",
+                    "x-ms-dynamic-values": {
+                      "operationId": "GetContacts",
+                      "value-path": "Phone",
+                      "value-collection": "Data",
+                      "value-title": "Name"
+                    },
+                    "description": "Choose Multiple Contact Names from Dropdown"
+                  },
+                  "description": "To"
+                },
+                "Body": {
+                  "type": "string",
+                  "description": "Message body",
+                  "title": "Text Message"
+                },
+                "ReplySTOPToOptOut": {
+                  "type": "boolean",
+                  "description": "IF \u201cYes\u201d THEN populate \u201cSTOP to opt out\u201d in the end of text message, and the contact can unsubscribe to text messages by replying 'STOP' to the text message. If the user replies to the text message with 'START', he/she gets subscribed back to the service again and starts receiving the text messages again. \u2022 IF \u201cNo\u201d THEN do not populate the unsubscribe text in the end of the text message.",
```

Comment:
@MohsinAQ Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.